README.rst

```diff
@@ -60,13 +60,13 @@ Other
    supported using the `Adafruit CircuitPython SD
    library <https://github.com/adafruit/Adafruit_CircuitPython_SD>`__)
 -  `Arduino Zero <https://www.arduino.cc/en/Main/ArduinoBoardZero>`__
+-  `Arduino MKR Zero <https://store.arduino.cc/arduino-mkrzero>`
 
 "Third-party" or "non-Adafruit" boards
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 -  `Electronic Cats Meow Meow <https://electroniccats.com/gomeow/>`__
--  `Electronic Cats CatWAN USB Stick <https://electroniccats.com/producto/catwan_usb_stick/>`__ 
```

Comment:
The trailing __ is needed for RST links here and above. You can see the output on GitHub by clicking View File: https://github.com/adafruit/circuitpython/blob/f8ded46ad69ebba2de5b791bde16c50c4d2b1d1b/README.rst

Reply:
Will do. I'll cancel the PR and re-submit with the test code in the other repo. Thanks!